# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.cpp
# Commit: f907e3b915ed
# Full Hash: f907e3b915ede8be7e3efc8888ea26fb095a331e
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p6 - Move nsPrintData::mPrintDocList to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146978
# ==============================================================================

diff -r 4e05361c6b08 -r f907e3b915ed layout/printing/nsPrintData.cpp
--- a/layout/printing/nsPrintData.cpp	Sun May 22 09:37:57 2022 +0000
+++ b/layout/printing/nsPrintData.cpp	Sun May 22 09:37:57 2022 +0000
@@ -24,7 +24,7 @@
 //-- nsPrintData Class Impl
 //---------------------------------------------------
 nsPrintData::nsPrintData(ePrintDataType aType)
-    : mType(aType), mPrintDocList(0), mOnStartSent(false), mIsAborted(false) {}
+    : mType(aType), mOnStartSent(false), mIsAborted(false) {}
 
 nsPrintData::~nsPrintData() {
   // Only Send an OnEndPrinting if we have started printing